README.md

```diff
@@ -1 +1,113 @@
+
+### How to create and submit a new proposal
+
+1. **First, check if a proposal for your idea already exists.** You can do this by looking at the [pull requests in this repo](https://github.com/amyleadem/uswds-proposals/pulls). If a proposal for your idea already exists, that’s great! You can contribute to the conversation by adding comments, information, and voting. You can learn more about [how to contribute to an existing proposal](#contributing-to-an-existing-proposal) in the next section.
+
+1. **If a proposal doesn't exist, it's time to create one.** Here are the steps to do that:
+    1. **Create a new branch.** [Create a new branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository) for your proposal named: `[proposed component name]-component`. For example, if you wanted to propose adding a loader component, you would name the branch `loader-component`.
+    1. **Copy the proposal template.** In your new branch, create a file in the `proposals` directory called `[component name]-component.md`. For example, `loader-component.md`. Copy the contents of [proposal-template.md](https://github.com/amyleadem/uswds-proposals/blob/add-readme-and-template/proposals/proposal-template.md) into this file.
+    1. **Fill out the template to the best of your ability.**  An asterisk (*) in the template indicates a required field. All required fields must be complete before your proposal can be considered for the Design System. However, to initially submit a proposal you must only complete the “Summary” and “Motivation” sections. You or someone else can finish the rest later.
+    1. **Open a pull request.** [Open a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) (PR) with the title: `Proposal: [Component name]`. For example, `Proposal: Loader component`. Once your PR is open, it will receive a "Status: Submitted" label and will be available for voting, comments, and additions.
+
+1. **Get the conversation started.** Once your PR has been opened, invite others to participate in the discussion and add information. Use the [uswds-public Slack channel](https://gsa-tts.slack.com/archives/C3F14AHSQ) or reach out to peers to solicit feedback and support.
+
+## Contributing to an existing proposal
+
+Every proposal can benefit from community input. If you would like to help with existing proposals, we encourage you to contribute by:
+
+- [Voting for proposals](#voting-for-a-proposal)
+- [Adding information, research, and solutions](#adding-information-to-an-existing-proposal)
+- [Sharing ideas and asking questions](#sharing-opinions-and-asking-questions)
```

Comment:
As a start, I re-ordered the content so that the README items all come before the link to the proposal. Open to suggestions about other ideas for what to do with this section. Wondering if it should be more strictly a table of contents for the README.